tensorboard/data/experimental/base_experiment.py

```diff
@@ -45,7 +45,12 @@ def get_scalars(
           pivot: Whether to returned DataFrame will be pivoted (via pandas’
             `pivot_data()` method to a “wide” format wherein the tags of a
             given run and a given step are all collected in a single row.
-            If not provided, defaults to `True`.
+            Setting `pivot` to `True` stipulates that the sets of step values
+            are identical among all tags of the experiment (after any run and
```

Comment:
Is this among all tags or among all time series?

Reply:
Good question. It requires that in each run, all tags have the same set of steps. The steps may differ between to runs for the same tag. I clarified that in the doc string here.